XIVComboVX/Combos/WHM.cs

```diff
@@ -76,6 +79,24 @@ internal class WhiteMageStone: CustomCombo {
 
 		if (Common.CheckLucidWeave(CustomComboPreset.WhiteMageLucidWeave, level, Service.Configuration.WhiteMageLucidWeaveManaThreshold, actionID))
 			return Common.LucidDreaming;
+		
+		if (IsEnabled(CustomComboPreset.WhiteMageDotRefresh))
+		{
+			uint dotForLevel = level switch {
+				>= WHM.Levels.Dia => WHM.Dia,
+				>= WHM.Levels.Aero2 => WHM.Aero2,
+				>= WHM.Levels.Aero => WHM.Aero,
+			};
```

Comment:
All of this can be replaced by a call to `OriginalHook(WHM.Aero)` since the game natively upgrades Aero -> Aero 2 -> Dia at the appropriate levels. It would probably be best to strip these five lines and simply `return OriginalHook(WHM.Aero)` in the conditional below.

Reply:
Oh sweet. I didn't know that method existed. Ty!

Reply:
Made a quick change. Unsure if there's a way to map spell -> debuff easier than the remaining switch

Reply:
Unfortunately, there isn't. The game has no functionality like that AFAIK, so it has to be done manually.